agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AiConfigCustomizer.java

```diff
@@ -53,22 +53,41 @@ public Map<String, String> apply(ConfigProperties otelConfig) {
         configuration.preview.captureHttpClientHeaders.responseHeaders);
 
     // enable capturing all mdc properties
```

Comment:
Coud you gather together all the `if(configuration.instrumentation.log4jAppender.enabled)`?

Reply:
it's grouped based on comments.